### PR TITLE
GOA - Include search;save 

### PR DESCRIPTION
**Pull request by renanberto**

### CODE REVIEW

**Comment by renanberto** [comment]
Please, after validation perform commits squash.

File: deployments/production/deployment.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+      application: mole-${DEPLOYMENT}
+  template:
+    metadata:
+      labels:
+        application: mole-${DEPLOYMENT}
+        deployment: ${DEPLOYMENT}
+    spec:
+      containers:
+        - name: mole-${DEPLOYMENT}
+          image: "registry.agilepromoter.com/iat/mole"
```
Comment (lucasmdrs) [inline deployments/production/deployment.yml]: use the `involvestecnologia` docker register

File: pkg/mongodb/oplog.go [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+	startTime, err := o.storage.StartTime()
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	cursor := o.oplogReader.Read(startTime)
+
+	for cursor.Next(context.TODO()) {
+		if err := cursor.Decode(&oplog); err != nil {
+			o.sendMsg("mole", "goa_logs", "#### [ MOLE ] Error to decode cursor", err.Error())
```
Comment (lucasmdrs) [inline pkg/mongodb/oplog.go]: `goa_logs` is hardcoded

File: pkg/storage/elasticsearch.go [hunk 3]
```diff
@@ -0,0 +1,152 @@
...
+	}
+}
+
+func (e *elasticsearch) SaveBatch(oplog models.Oplog) error {
+	resource, err := e.prepare(oplog)
+	if err != nil {
+		return err
+	}
+
+	meta := []byte(fmt.Sprintf(`{ "index" : { "_id" : "%d" } }%s`, oplog.HistoryID, "\n"))
```
Comment (lucasmdrs) [inline pkg/storage/elasticsearch.go]: `\n` inserted during string format

File: pkg/mongodb/oplog.go (hunk 2 above)
Comment (renanberto) [reply pkg/mongodb/oplog.go]: we will find a better way to send this information

File: deployments/production/deployment.yml (hunk 1 above)
Comment (renanberto) [reply deployments/production/deployment.yml]: It will be removed

File: pkg/storage/elasticsearch.go (hunk 3 above)
Comment (renanberto) [reply pkg/storage/elasticsearch.go]: can you explain it better?
we are sending a string to break the line, interpreted by the index

Comment (lucasmdrs) [reply pkg/storage/elasticsearch.go]: nevermind, the backtick would interpret the `\n` as a literal value.

**Pull request merged**